examples/android/android-studio/Cukeulator/README.md

```diff
@@ -0,0 +1,60 @@
+## Cukeulator Example Test
+This is the example test-project for the Cukeulator app for Android Studio (Beta) 0.8.14.
+
+### Setup
+Features must be placed in `assets/features/`. Subdirectories are allowed.
+
+The `app/libs` should contain `cucumber-android-*.jar`. Please build it following [cucumber-jvm/android]
```

Comment:
Thanks, Sabastian! I wasn't aware of this option. It works for me. I will
update the change.

- Mykola

On Sat, Jan 3, 2015 at 7:10 AM, Sebastian Gröbler <notifications@github.com>
wrote:

> In examples/android/android-studio/Cukeulator/README.md
> <https://github.com/cucumber/cucumber-jvm/pull/816#discussion-diff-22432069>
> :
>
> > @@ -0,0 +1,60 @@
> > +## Cukeulator Example Test
> > +This is the example test-project for the Cukeulator app for Android Studio (Beta) 0.8.14.
> > +
> > +### Setup
> > +Features must be placed in `assets/features/`. Subdirectories are allowed.
> > +
> > +The `app/libs` should contain `cucumber-android-*.jar`. Please build it following [cucumber-jvm/android]
>
> why not adding it through the build.gradle as well?
>
> e.g. androidTestCompile 'info.cukes:cucumber-android:1.1.8@jar'
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/cucumber/cucumber-jvm/pull/816/files#r22432069>.
>